pkgs/development/python-modules/roundrobin/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "roundrobin";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "linnik";
+    repo = pname;
+    rev = "5a5b34f2cd3ab8aa6d4135b31ab512b2d9338c19";
+    sha256 = "sha256-h+GcsHM0D4uBMcOng6mioC5o5AWbegiGqGYtuPoduM4=";
+  };
+
+  checkPhase = ''
+    ${python.interpreter} -m unittest discover
```

Comment:
following up to my last comment: i learned in a different discussion about a way to do the `checkPhase` for python packages. i've just implemented that here.